tools/windows_portable/generate_update_keypair.ps1

```
@@ -0,0 +1,241 @@
+  $dp = Convert-ToBase64 -Bytes $Parameters.DP
+  $dq = Convert-ToBase64 -Bytes $Parameters.DQ
+  $inverseQ = Convert-ToBase64 -Bytes $Parameters.InverseQ
+  $d = Convert-ToBase64 -Bytes $Parameters.D
+  return (
+    "<RSAKeyValue><Modulus>$modulus</Modulus><Exponent>$exponent</Exponent>" +
+    "<P>$p</P><Q>$q</Q><DP>$dp</DP><DQ>$dq</DQ>" +
+    "<InverseQ>$inverseQ</InverseQ><D>$d</D></RSAKeyValue>"
+  )
+}
+
+function Set-PrivateFileAcl([string]$PathValue) {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Set-PrivateFileAcl' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)